src/llvm-coverage-instrumentation.md

```diff
@@ -221,15 +221,15 @@ substitution combinations), `mapgen`'s `finalize()` method queries the
 and `CodeRegion`s; and calls LLVM codegen APIs to generate
 properly-configured variables in LLVM IR, according to very specific
 details of the [_LLVM Coverage Mapping Format_][coverage-mapping-format]
-(Version 4).[^llvm-and-covmap-versions]
+(Version 6).[^llvm-and-covmap-versions]
 
 [^llvm-and-covmap-versions]: The Rust compiler (as of
```

Comment:
Could you add a machine-checkable date comment?

```suggestion
[^llvm-and-covmap-versions]: The Rust compiler (as of <!-- date: 2021-11 -->)
```

Reply:
I made this change in the revised PR #1267.